MovieTime/MovieTime/Model/MTModelMovieDBGenre.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MTModelMovieDBGenre.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 10/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import Foundation
+
+struct MTModelMovieDBGenre: Codable {
+    var id: Int?
+    var name: String?
+}
+
+// Sample data
```

Comment:
it's nice that you worried about adding sample data to the classes, however they're just comments, so if they get outdated, nothing ensures that people will care about them. Ideally they would be put into JSONs and be part of your unit tests, so if the code or the API update, the sample data files will have to reflect that